src/administrative-unit/administrative-unit.service.ts

```diff
@@ -31,6 +31,14 @@ export class AdministrativeUnitService {
     }
   }
 
+  async getAllDistricts() {
+    try {
+      return await this.repoDistrict.find();
+    } catch (error) {
+      throw new HttpException('Not found districts', HttpStatus.NOT_FOUND);
+    }
```

Comment:
find array ko cần throw ra exception not found vì luôn trả về mảng rỗng nếu ko có